### PR TITLE
Moving invest capital form into Atlas

### PR DESCRIPTION
**Pull request by webbyfox**
To do (delete all that do not apply):

 - [ ] Change has a jira ticket that has the correct status.
 - [ ] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [ ] (if adding CSS) CSS have been compiled.
 - [ ] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [ ] (if changing content) Locale files have been updated and translated strings have been added if available.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags
 - [ ] (if changing the UI) Add a printscreen to the PR

### CODE REVIEW

File: conf/url_redirects.py [hunk 1]
```diff
@@ -10,6 +10,13 @@
         r'^international/invest[/]*$',
         QuerystringRedirectView.as_view(pattern_name='atlas-home'),
     ),
+    url(
+        # Redirect the old capital invest contact form to to atlas
+        r'^international/content/capital-invest/contact[/]*$',
```
Comment (bobmeredith) [inline conf/url_redirects.py:15]: Being a pedant, I think the regexp should be like '.../contact[/]?$' or even '.../contact\/?$' as I don't see why we would want to match multiple slashes.
It's no biggie though as it looks like this path is consistent with the others.

Comment (webbyfox) [reply conf/url_redirects.py:15]: There was the ticket for allowing redirect for multiple slashes - hence this was introduced before.

Comment (bobmeredith) [reply conf/url_redirects.py:15]: Fair enough

**Review by bobmeredith — APPROVED** [review]
LGTM

**Review by froddd — APPROVED** [review]
LGTM

**Pull request merged**